File: PPA/WriteFile.java
import java.io.*;

class WriteFile
{
    public static void main(String arg[])// throws Exception
    {
        try
        {
            String Data="Marvellous InfoSystems";
            byte Arr[]=Data.getBytes();  //char is 2 bytes so converting it to 1 bytes as file can read only 1 byte at a time
            FileOutputStream fobj=new FileOutputStream("PPA.txt");

            fobj.write(Arr);
        }
        catch(Exception obj)
        {}      
    }
}

File: PPA/WriteFile.c
#include<stdio.h>
#include<fcntl.h>   //File Control (has file control operations)
#include<unistd.h>  //Universal Standards
#include<string.h>
int main()
{
    int fd=0;  //File descriptor

    char Arr[]="Marvellous Infosystems";
    int Ret=0;

    fd=open("Marvellous.txt",O_RDWR);

    Ret=write(fd,Arr,strlen(Arr));
    //  write(Kashat lihayha,Kay lihaycha,Kiti lihaycha(22 letters in our case))

    printf("%d bytes gets written in the File\n",Ret);
    
    close(fd);
    return 0;

    
}

//O_RDONLY : Read
//O_WRONLY : Write
//O_RDWR   :Read+Write
